WordPress/src/main/java/org/wordpress/android/ui/prefs/EmptyViewRecyclerView.java

```diff
@@ -0,0 +1,71 @@
+package org.wordpress.android.ui.prefs;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+import android.view.View;
+
+/**
+ * RecyclerView with setEmptyView method.
+ */
+public class EmptyViewRecyclerView extends RecyclerView {
+    private View emptyView;
```

Comment:
Let's add `m` prefix to `emptyView` to be consistent with AOSP naming convention :)